### ISSUE TITLE
The type of nextPageToken and prePageToken parameters in CursorPaginatedNonFungibleBalanceWithTypes is not `string` but `string?`

### ISSUE THREAD

**Issue by YuheiNakasaka** [line/line-blockchain-developers-sdk-js#56]
### Default information
* lbd-sdk-js version: `1.3.0`
* nodeJs version: `v16.11.1`
* module-type: `commonJs`

### Description of the bug
```ts
export class CursorPaginatedNonFungibleBalanceWithTypes {
  constructor(
    readonly list: Array<NonFungibleBalanceWithType>,
    readonly prePageToken: string,
    readonly nextPageToken: string,
  ) { }
}
```

The type of `nextPageToken` and `prePageToken` parameters in `CursorPaginatedNonFungibleBalanceWithTypes` is `string` now. But [This api](https://docs-blockchain.line.biz/api-guide/category-users/retrieve#v1-users-userId-item-tokens-contractId-non-fungibles-with-type-get) is returning `nextPageToken` and `prePageToken` response  as `string?`.

I don't know whether this difference is a bug of the document or the implementation.

#### Example codes
If the api returns status 1000 and `responseData.list` is **not empty**, then returns `''`(empty string) or `'xxxxx'`(page token) as `nextPageToken` and `prePageToken` responses. But it returns status 1000 and `responseData.list` is **empty**, then returns `null` as `nextPageToken` and `prePageToken` responses. 

```js
const sdk = require("@line/lbd-sdk-js");
const load = require("ts-dotenv").load;

const env = load(
    {
        "HOST_URL": String,
        "SERVICE_ID": String,
        "SERVICE_API_KEY": String,
        "SERVICE_API_SECRET": String
    }, {
    "path": "./sdk.env"
}
);

const client = new sdk.HttpClient(env.HOST_URL, env.SERVICE_API_KEY, env.SERVICE_API_SECRET);

const request = new sdk.CursorPageRequest();
client.nonFungibleTokenBalancesWithTypeOfUser("USER_ID", "CONTRACT_ID", request).then(response => {
    console.log("responseData", response.responseData);
});
```

#### Error logs
Nothing

### Anything else we need to know
Nothing

**Comment by ryukato** [comment]
Oh, I guess that is a sort of mistake on this sdk, I will check it again and let you know when it is fixed. Thank you.

**Comment by ryukato** [comment]
PTAL https://github.com/line/line-blockchain-developers-sdk-js/pull/59

**Comment by ryukato** [comment]
I will close this because this issue has been fixed by #59 and will be released soon.
